CGPs/cgp-0155.md

```diff
@@ -3,9 +3,9 @@ cgp: 155
 title: Add CeloToken and LockedCelo to the registry
 date-created: 2024-11-19
 author: Sebastien Benoit (@soloseng), Martín Volpe (@martinvol)
-status: DRAFT
-discussions-to:
-governance-proposal-id:
+status: Poposed
```

Comment:
Please correct spelling mistake so the CI runs clean and downstream dependant applications do not have errors